cli/dcoscli/node/main.py

```diff
@@ -748,20 +750,25 @@ def _ssh(leader, slave, option, config_file, user, master_proxy, command):
     if command is None:
         command = ''
 
-    master_public_ip = dcos_client.metadata().get('PUBLIC_IPV4')
-    if master_proxy:
+    proxy_ip = proxy_ip or config.get_config_val('core.dcos_proxy_ip')
```

Comment:
Yeah. I can take it out. I don't know if it changes your opinion, but the reason I did this was less for the config file setting and more for the environment variable, which comes for free. I guess I could have it read `os.environ`.